app/gui2/src/components/GraphEditor/nodeCreation.ts

```diff
@@ -1,10 +1,6 @@
 import type { Pattern } from '@/util/ast/match'
-import type { Vec2 } from '@/util/data/vec2'
 
 interface AllNodeCreationOptions {
-  /** Where to place the node; if a value is specified, it will be used exactly.
-   *  If unspecified, a location will be chosen based on the source node. */
-  position?: Vec2 | undefined
```

Comment:
Why this short file is something different from `nodeCreation` composable?

Reply:
This is a high-level API that provides a simpler interface than the composable for use from widgets and visualizations.